hr_af_holidays/__init__.py

```diff
@@ -0,0 +1,3 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from . import models
```

Comment:
Remove this line and remove the models folder from the module. Keep this __init__.py file in the root